assets/dev/js/admin/admin.js

```diff
@@ -407,14 +407,15 @@ import { showJsonUploadWarningMessageIfNeeded } from 'elementor-utils/json-uploa
 				$( '#elementor-import-template-area' ).toggle();
 			} );
 
-			let messageShown = false;
+			let warningMessageShown = false;
+			let notEnabledMessageShown = false;
```

Comment:
Not clear what it represents maybe a different name?

Reply:
Done.